src/fonts-sidebar/index.js

```diff
@@ -0,0 +1,182 @@
+		},
+		0
+	);
+
+	const getFileSize = ( url ) => {
+		return fileSizes[ url ] ? bytesToSize( fileSizes[ url ] ) : null;
+	};
+
+	const totalSize = bytesToSize(
+		flatfontsOutline.reduce( ( acc, face ) => {
+			return acc + parseInt( fileSizes[ face.src ] );
+		}, 0 )
+	);
+
+	return (
+		<div className="sidebar">
+			<div className="sidebar-container">
+				<div className="variants-outline">
+					<h2>{ title }</h2>
+					<h3>{ fontsOutline.family }</h3>
+
+					{ !! fontsOutline && (
+						<>
+							<div className="variants-list">
+								<div className="content">
+									{ Object.keys( fontsOutline ).map(
+										( key, i ) => (
+											<div
+												className="variants-family"
+												key={ `variants-family-${ i }` }
+											>
+												<div className="header">
+													<span className="name">
+														{
+															fontsOutline[ key ]
+																.family
+														}{ ' ' }
+													</span>
+													<div>
+														<span className="variants">
+															({ ' ' }
+															{
+																fontsOutline[
+																	key
+																].faces.length
+															}{ ' ' }
+															{ _n(
+																'Variant',
+																'Variants',
+																fontsOutline[
+																	key
+																].faces.length,
+																'create-block-theme'
+															) }{ ' ' }
+															)
+														</span>
+														<Button
+															icon={ trash }
+															iconSize={ 15 }
+															isSmall
+															onClick={ () =>
+																handleDeleteFontFamily(
+																	fontsOutline[
+																		key
+																	].family
+																)
+															}
+														/>
+													</div>
+												</div>
+												{ fontsOutline[ key ].faces.map(
+													( face, ii ) => (
+														<div
+															className="variant-row"
+															key={ `selected-variant-${ ii }` }
+														>
+															<div className="variant">
+																{ face.weight }{ ' ' }
+																{ face.style }
+															</div>
+															<div className="size">
+																{ getFileSize(
+																	face.src
+																) }
+															</div>
+															<div>
+																<Button
+																	onClick={ () =>
+																		handleDeleteFontFace(
+																			fontsOutline[
+																				key
+																			]
+																				.family,
+																			face.weight,
+																			face.style
+																		)
+																	}
+																	icon={
+																		trash
+																	}
+																	iconSize={
+																		15
+																	}
+																	isSmall
+																/>
+															</div>
+														</div>
+													)
+												) }
+											</div>
+										)
+									) }
+								</div>
+							</div>
```

Comment:
because of too much nesting code isn't very readable. created #265 to address the issue separately.

Reply:
Yes, the code formatter is not working great in some cases.